s3_file_uploads/serializers.py

```diff
@@ -28,3 +29,11 @@ class Meta:
             'file_path',
             'file_key'
         ]
+
+class AccessControlListSerializer(serializers.Serializer):
+    acl = serializers.CharField(required=False)
+
+    def validate_acl(self, value):
```

Comment:
You can specify a `choices` argument to `CharField`, which removes the need to implement it outselves

Reply:
choices is not an option for Charfield - tried this

Reply:
Agreed as discussed and updated